### DISCUSSION TITLE
Planning support for K2 Plus?

### DISCUSSION THREAD

**Discussion by Ltek** [Guilouz/Creality-Helper-Script-Wiki#671] [General]
@Guilouz are you planning to support K2 Plus?  
https://www.creality.com/products/creality-k2-plus-cfs-combo

**Comment by Ltek** [comment]
@Guilouz assuming no reply means you do not plan to do K2 Plus.

**Comment by asgoldenberg** [comment]
@Guilouz I deeply appreciate all the work you've done so far -- bien fait. Would it be helpful that I "give back" by conducting some early testing on a K2? What are your intentions here, so I may collaborate? Our first blocker would be root access, which I now understand [is available](https://github.com/Guilouz/Creality-Helper-Script-Wiki/discussions/688#discussioncomment-11181371
).

**Comment by Guilouz** [comment]
Currently the script is not compatible with K2, the OS is slightly different.

I'm waiting for Creality to send it to me. Normally in December.

**Reply by Ltek** [reply]
I've printed a few different items (some functional, some decorative) on my K2 Plus... and WOW is it already 5x nicer/better than my K1 Max was with all the tuning/tweaking. 

The main improvement IMO would be if KAMP worked with the Print Calibration (Lidar)

I've been using Creality Print 5.1 (CP5) with the Print Calibration and its been nearly perfect. Unless Orca adds integration with the Lidar, I'm going to stick with CP5

**Reply by asgoldenberg** [reply]
Super interesting and helpful to hear--I'll look to accumulate the K2 over the K1 Max based on your feedback. Which means, I would have one to help test, eventually.

Does the K2 tend to work out of the box? On my K1C out-of-box configuration, the touchscreen would lock up after start of print _every time_.

**Reply by ccollinscj** [reply]
I am waiting to be be able to root the K2 as well,  like to use the tools to level the bed like [screws_tilt_adjust]  which doesnt seem to work on the K2

**Reply by soundstorm** [reply]
The Tina Linux based on OpenWRT has neither git, nor wget or curl. Interesting...

**Reply by Guilouz** [reply]
This is not a problem. Script for k2 is in progress

**Reply by Iris-pro** [reply]
@Guilouz So excited hearing this.🎉🎉🎉

**Reply by asgoldenberg** [reply]
@Guilouz Like [Iris-pro](https://github.com/Iris-pro), I appreciate hearing this. Now that I know the K2 script is in progress, I've ordered a K2.

**Reply by TheTherminator** [reply]
@Guilouz Awesome, that will improve the K2 and make it finally error free. Way to go 👍🏼

**Reply by belguinha** [reply]
Good work! I'm also with one k2, once again, thanks by all the work you made for k series.

**Reply by Barconero** [reply]
> This is not a problem. Script for k2 is in progress

How's "progress" going? ^_^

**Comment by ricklee1210** [comment]
I got my K2 today and it is indeed amazing. I have the K1 max and after tweaking and modding the crap out of it the K2 prints better out of the box. Can’t wait for the script, I’ll be donating for sure.

**Comment by ABBMax** [comment]
Any News regarding Creality K2 Custom firmware ? is there a estimated time plan ?

**Comment by apache07x** [comment]
@Guilouz are there any updates on the helper script for the K2 series?

**Comment by Styxiex** [comment]
Sorry to push you, but I really want adaptive bed meshing for the K2 plus, any chance you could update us on your progress? Sorry to bother once again, goodluck coding!

**Comment by Schulli73** [comment]
THE link is Dead  https://github.com/Guilouz/Creality-Helper-Script-K2-Wiki

**Reply by Styxiex** [reply]
What was there before it was dead?

**Reply by LifeDay** [reply]
Something like a wiki template with the words "coming soon"

**Reply by VashBaldeus** [reply]
There was no updates from the developer regarding K1 CFS/K2 support

**Reply by apache07x** [reply]
Is he still doing anything? Unlike him not to respond to anyone much less for this long with no response.

**Comment by GromovPeter** [comment]
i have first version of adaptive mesh for k2 pro you can use it for k2 series
https://github.com/GromovPeter/K2PRO/blob/main/README.md

**Comment by sw3defy** [comment]
> @Guilouz  are you planning to support K2 Plus? https://www.creality.com/products/creality-k2-plus-cfs-combo

@Ltek I have created a helper script for the K2 Plus, you find it here https://github.com/sw3defy?tab=repositories

**Reply by KaraokeAmerica** [reply]
Is it ONLY for the Plus or does it work on the regular K2 Combo and/or the Pro?

**Reply by sw3defy** [reply]
As of now it is ONLY for the K2 Plus Combo and standalone K2 Plus.

**Reply by Barconero** [reply]
May be some points of your helper are compatible with Pro? Like Moonraker, Fluidd and Belt calibration?
